src/utils/resources/template/hooks/useProcessedTemplate.ts

```diff
@@ -11,7 +11,7 @@ import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
  */
 export const useProcessedTemplate = (
   template: V1Template,
-  namespace: string,
+  namespace = 'default',
```

Comment:
WDYT of creating a constant file under `/src/utils` for the `default` namespace? Also for `getTemplateVMWithNamespace`

Reply:
I saw we do have one under catalog directory, I'll move it